### ISSUE TITLE
[INVERSE_FUNCTION_PREFERRED] should not replace !ClosedRange.isEmpty()

### ISSUE THREAD

**Issue by illarionov** [saveourtool/diktat#1699]
## Describe the bug
Given the following code:
```kotlin
val range: ClosedRange<Int> = 0..1
check(!range.isEmpty())
```
Diktat suggests replacing this with the nonexistent `ClosedRange.isNotEmpty()` function:
```
[INVERSE_FUNCTION_PREFERRED] it is better to use inverse function: isNotEmpty() instead of !isEmpty() (diktat-ruleset:inverse-method)
```

## Expected behavior
Rule should not be triggered when the `isNotEmpty` function does not exist.

## Environment information
* diktat version: 1.2.5
* how is diktat run (CLI, plugin, etc.): CLI

**Comment by akuleshov7** [comment]
Haha, nice catch, thank you! We will fix it. 
But actually strange that they did not implement `isNotEmpty()` function, may be need to contribute to the stdlib 🤔

**Comment by akuleshov7** [comment]
As a quick workaround you can use `@Suppress("INVERSE_FUNCTION_PREFERRED") `
